exercises/practice/rest-api/.docs/instructions.append.md

```diff
@@ -0,0 +1,9 @@
+# Instructions append
+
+## Implementation
+
+Implement the `get` and `post` methods from the `RestAPI` class.
+
+- no actual HTTP server should be implemented, just the handler functions should be written
+- an in-memory object can serve as "database"
+- the constructor function needs to accept a seed for the database as argument (and set up a sensible default if nothing was passed)
```

Comment:
Could you try to convert these into your own proper sentences? Currently it looks weird because they address different things and the relation to the sentence above is unclear.